src/ADNLPProblems/genbroyden.jl

```diff
@@ -0,0 +1,40 @@
+export genbroyden
```

Comment:
```suggestion
export genbroydentri
```
maybe a better name, what do you think?